proto/buf/alpha/registry/v1alpha1/repository.proto

```diff
@@ -54,11 +54,6 @@ message Repository {
   // url is the user configurable URL in the description of the repository,
   // always included the scheme and will not have a #fragment suffix.
   string url = 12;
-  // license_spdx_id is the current license of the repository, based on the
-  // license file pushed, which should be one of the identifier defined in
-  // https://spdx.org/licenses, and will be empty if the license file is not
-  // presented or cannot be classified into a known license.
-  string license_spdx_id = 13;
```

Comment:
we might want to reserve the field name and number for this, but as this field is added very recently and we never actually use it, it should be fine to just remove it

Reply:
If it has not been used, I think the breaking change is reasonable here.